sycl/unittests/helpers/CommandSubmitWrappers.hpp

```diff
@@ -0,0 +1,36 @@
+//==-- CommandSubmitWrappers.hpp ---  -----==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <sycl/event.hpp>
+#include <sycl/handler.hpp>
+
+using namespace sycl;
```

Comment:
I'd personally avoid using `using namespace sycl` here as it will pollute the global namespace of every test that includes this header file.
How about we do the following instead:
```
namespace sycl {

template <typename KernelName, typename KernelType>
event single_task_wrapper(bool ShortcutSubmitFunction, queue &Q,
                          const KernelType &KernelFunc) {
...

} // namespace sycl
```